src/tests/example.spec.js

```diff
@@ -24,4 +26,56 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    test('check backward sorting', async ({ loginPage, inventoryPage }) => {
```

Comment:
Could you implement all the possible sorting tests?
Consider using [parameterized tests](https://playwright.dev/docs/test-parameterize)